clipper_admin/clipper_admin/sample_config.yaml

```diff
@@ -0,0 +1,23 @@
+Model1:
+  obj_type: "model"
+  input_type: "integers"
+  version: "1.2.7"
+  image: "test-model:1"
+
+Application1:
+  obj_type: "application"
+  input_type: "doubles"
+  slo: 10000
+  default_value: "not received"
+
+Model2:
+  obj_type: "model"
+  input_type: "floats"
+  version: "1.2.7"
+  image: "train-model:2"
+
+Application2:
```

Comment:
Great work! It's a better practice to keep the key of dictionary to be static/immutable/predictable: checkout https://github.com/ucbrise/clipper/blob/develop/clipper_admin/clipper_admin/kubernetes/query-frontend-deployment.yaml